docs/BaremetalHost_ProvisioningState.dot

```diff
@@ -36,6 +36,7 @@ digraph BaremetalHost {
     Available [shape=doublecircle]
     Available -> Provisioning [label="NeedsProvisioning()"]
     Available -> Preparing [label="saveHostProvisioningSettings()"]
+    Available -> Preparing [label="getHostFirmwareSettings()"]
```

Comment:
Why are there 2 edges from Available->Preparing now?

Reply:
@zaneb I believe due to https://github.com/metal3-io/baremetal-operator/commit/cd3340b20de240feb1566432def9fc6b678f7626#diff-eeed3703d04e4c23a7d7af8cd0b7931b6b7990f23d826c49bdbc31c5f0a50291 since the other one `saveHostProvisioningSettings` was already there https://github.com/metal3-io/baremetal-operator/issues/1122#issue-1244956859